azkaban-web-server/src/main/java/cloudflow/models/ExecutionNodeResponse.java

```diff
@@ -0,0 +1,265 @@
+package cloudflow.models;
+
+import static com.google.common.base.Preconditions.checkState;
+import static java.util.Objects.requireNonNull;
+
+import com.sun.istack.internal.Nullable;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.codehaus.jackson.annotate.JsonIgnore;
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.codehaus.jackson.annotate.JsonPropertyOrder;
+import org.codehaus.jackson.map.annotate.JsonSerialize;
+
+@JsonPropertyOrder({"name", "type"})
+@JsonSerialize(include=JsonSerialize.Inclusion.NON_NULL)
+public class ExecutionNodeResponse {
+
+  @JsonProperty("name")
+  private final String nodeId;  // maps to ExecutableNode.id
+
+  @JsonProperty("type")
+  private final ExecutionNodeType nodeType;
+
+  private final long startTime;
+  private final long endTime;
+  private final String status;
+
+  @JsonIgnore
+  private final long updateTime;
+
+  @Nullable
+  private final FlowBasicResponse flowInfo;
+
+  @JsonIgnore
+  @Nullable
+  private final String condition;
+
+  @JsonIgnore
+  @Nullable
+  private final String nestedId;
+
+  @JsonIgnore
+  @Nullable
+  private final List<String> inputNodeIds; // maps to ExecutableNode.inputNodes
```

Comment:
I believe this field is also not used.

Reply:
I decided to remove some of these fields only from the json output as I suspect they will need to be added sooner than later as  users may be quite interested in fields like input-nodes, especially if we decide to make them visible through an `expand` like option in the near future. 
Having this additional info available doesn't hurt.